state-chain/pallets/cf-flip/src/lib.rs

```diff
@@ -620,6 +622,20 @@ impl<T: Config> cf_traits::Funding for Pallet<T> {
 		T::OnAccountFunded::on_account_funded(account_id, Self::balance(account_id));
 		Ok(())
 	}
+
+	fn try_transfer_funds_internally(
+		amount: Self::Balance,
+		from: &Self::AccountId,
+		to: &Self::AccountId,
+	) -> Result<(), DispatchError> {
+		ensure!(from != to, Error::<T>::CanNotTransferToSelf);
+		if let Some(from_imbalance) = Pallet::<T>::try_debit(from, amount) {
+			from_imbalance.offset(Pallet::<T>::credit(to, amount));
+			Ok(())
```

Comment:
This should function (almost) exactly as if we called:
`try_initiate_redemption(from, amount)` -> `finalize_redemption(from)` -> `credit_funds(to, amount)`

So we need to call `T::OnAccountFunded::on_account_funded` for each account to trigger the callbacks for account balance changes.